wp-content/themes/goonj-crm/functions.php

```diff
@@ -182,6 +182,13 @@ function goonj_handle_user_identification_form() {
 			'not-inducted-for-dropping-center'
 		);
 
+		$institute_registration_individual_signup_form_path = sprintf(
```

Comment:
can we use a single name either `registration` or `sign up`. I think we should use just registration

Reply:
also I don't think we should create individual-sign-up as a child page of institute registration. they are independent

Reply:
sure will update